app/assets/stylesheets/sbstyles.scss

```diff
@@ -19,8 +22,51 @@ h2 {
 
 .top-bar {
   margin-bottom: 10px;
+  box-shadow: 0 1px $tertiary-green;
+  background: linear-gradient($primary-green, $secondary-green);
 }
 
 .button.tiny.radius {
   background-color: $review-button-color;
 }
+
+.bg {
+  min-height: 100%;
+  min-width: 1024px;
+  width: 100%;
+  height: auto;
+  position: fixed;
+  top: 56px;
+  left: 0;
+}
+
+.landing_wrapper {
+  display: block;
+  margin: auto;
+  top: 56px;
+  left: 0;
+  height: 400px;
+}
+
+.block {
+  background: white;
+  opacity: 0.5;
```

Comment:
Properties should be ordered background, height, left, opacity, position, top, width